### DISCUSSION TITLE
4.0.0rc1

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#27173] [General]
Hello Superset Community,

Superset 4.0.0 RC1 is up for testing. Check the [Testing a Release Candidate (RC)](https://github.com/apache/superset/wiki/Release-Process#testing-a-release-candidate-rc) section of our [Release Process](https://github.com/apache/superset/wiki/Release-Process) for information on how to test and report issues with the release candidate.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/4.0.0rc1/

The Git tag for the release:
https://github.com/apache/superset/tree/4.0.0rc1

The change log for the release:
https://github.com/apache/superset/blob/4.0.0rc1/CHANGELOG/4.0.0.md

The instructions for updating to the release:
https://github.com/apache/superset/blob/4.0.0rc1/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/4.0.0rc1'>4.0.0rc1</a>.</em>

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/27177 @sadpandajoe

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/27484, https://github.com/apache/superset/pull/27566

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/27310 @sadpandajoe

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/27361

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/27311 @sadpandajoe

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/27313

**Comment by sfirke** [comment]
#27374 ?  Someone suggests a code fix there to get reports working with Chrome.  I will note that I tested 4.0.0rc1-dev image which bundles Firefox and Geckodriver and confirmed that email reports work out of the box.

**Reply by michael-s-molina** [reply]
The original issue was closed. See https://github.com/apache/superset/issues/27374#issuecomment-1976858102

**Comment by sfirke** [comment]
~~Drill to detail broke in a regression, #27424 - I can screenshare or provide add'l details if needed~~ Closed issue as I cannot replicate.  Not sure what is/was happening but no need for Michael to investigate.  If others want to test drill to detail, that couldn't hurt.

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/27560

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/27577

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/issues/27549

**Reply by michael-s-molina** [reply]
Fix: https://github.com/apache/superset/pull/27576

**Comment by john-bodley** [comment]
https://github.com/apache/superset/issues/27581

**Reply by john-bodley** [reply]
Fix: https://github.com/apache/superset/pull/27470
